### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by ibrajkhan**
Assignment 2 completed

### CODE REVIEW

File: src/Assignment-2/assignment2.html [hunk 1]
```diff
@@ -1,40 +1,132 @@
...
+    <meta
+      http-equiv="Cache-Control"
+      content="no-cache, no-store, must-revalidate"
+    />
+    <meta http-equiv="Pragma" content="no-cache" />
+    <meta http-equiv="Expires" content="0" />
+    <script src="bundle.js" type="text/javascript"></script>
+  </head>
 
-<body>
```
Comment (chhaviteotia) [inline src/Assignment-2/assignment2.html:11]: line no. 11, 12 are not necessary.

File: src/Assignment-2/assignment2.html [hunk 2]
```diff
@@ -1,40 +1,132 @@
...
 
-	<!-- Start your code from here -->
-	
+      <!-- Start your code from here -->
 
-	<!-- End your code -->
+      <!-- End your code -->
+    </div>
 
-	</div>
```
Comment (chhaviteotia) [inline src/Assignment-2/assignment2.html:19]: extra tag,not required

**Review by chhaviteotia — COMMENTED** [review]
be mindful towards closing and opening of tags, rest seems fine to me.

**Review by chhaviteotia — APPROVED** [review]
place screenshot of output.

**Comment by SuneetKumar19** [comment]
Great work

**Pull request merged**